src/main/java/org/testng/ITestContext.java

```diff
@@ -81,7 +82,13 @@
   /** @return the current XmlTest. */
   XmlTest getCurrentXmlTest();
 
-  List<Module> getGuiceModules(Class<? extends Module> cls);
+  List<Module> getAllGuiceModules(Class<? extends Module> cls);
```

Comment:
Sorry, I didn't explain enough.
I think `getGuiceModules` is good for the old method but the new one should be named `getAllGuiceModules` because it is returning all modules without any filter

Reply:
@juherr - My bad. That was a bone headed mistake from my side. I ended up renaming the wrong method :(
Now fixed it.